miner/payload_building_test.go

```diff
@@ -82,7 +82,7 @@ func TestBuildPayload(t *testing.T) {
 }
 
 func TestPayloadId(t *testing.T) {
-	ids := make(map[string]int)
+	ids := make(map[string]int, 7)
```

Comment:
IMO micro-optimizations inside tests isn't worth it. I'd rather not touch it personally